onnxruntime/core/graph/graph.cc

```diff
@@ -3866,16 +3866,19 @@ Node& Graph::CreateFusedSubGraphNode(const IndexedSubGraph& sub_graph, const std
   std::vector<NodeArg*> output_args;
   std::unordered_map<std::string, int> input_indexes;
   std::unordered_map<std::string, int> output_indexes;
+  // If the subgraph we are going to build is the subgraph of the original graph and the NodeArgs of the outer scope values
+  // are defined in the top-level original graph. It needs to get those NodeArgs from ancestors.
+  bool check_ancestors = func_meta_def->check_ancestors ? true : false;
```

Comment:
Should this default to using GetNodeArgIncludingParentGraphs in all cases so we don't need a 'check_ancestors' property? 

I don't think that would change any existing behavior as the local node arg will always be preferred.

Reply:
I didn't notice GetNodeArgIncludingParentGraphs until you pointed it out.
Yes, I just changed it. Since GetNodeArgIncludingParentGraphs won't change any existing behavior but will address the issue for the case I mentioned.